File: Pizzeria.UI/pizzeria-ui/src/interfaces/Pizza.ts
import { PizzaIngredient } from "./PizzaIngredient"
import { PizzaPrice } from "./PizzaPrice"

export interface Pizza{
    id: number,
    name: string,
    publicImageId: string,
    displayOrder: number,
    pizzaIngredients: PizzaIngredient[],
    pizzaPrices: PizzaPrice[]
}

File: Pizzeria.Core/Entities/Pizza.cs
using Pizzeria.Core.Entities.Base;

namespace Pizzeria.Core.Entities
{
    public class Pizza : BaseEntity
    {
        public string Name { get; set; }
        public string PublicImageId { get; set; }
        public int? DisplayOrder { get; set; }

        public HashSet<PizzaIngredient> PizzaIngredients { get; set; }
        public HashSet<PizzaPrice> PizzaPrices { get; set; }
    }
}
